tests/test_hyper_dicer.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for the hyper_dicer tool.
+"""
+import json
+from os import path
+from pathlib import Path
+import pytest
+import responses
+
+from dicer.config import keycloak_config, transmart_config
+from dicer.hyper_dicer import HyperDicer
+from tests.mock_responses import POST_JSON_RESPONSES, GET_JSON_RESPONSES
+
+
+@pytest.fixture
+def mocked_responses():
```

Comment:
why not use the test mock server?

Reply:
I wanted to compare the two approaches: 1) mocking the server 2) mocking the behaviour of the requests package. And I tried to use `pytest` instead of `unittest`. Now that we have both, we can compare which we like more. I like the simplicity of mocking the requests.